server/vcs/provider/github/repo_fetcher.go

```diff
@@ -112,6 +119,10 @@ func (g *RepoFetcher) generateDirPath(repoName string) string {
 	return filepath.Join(g.DataDir, workingDirPrefix, repoName, uuid.New().String())
 }
 
+func (g *RepoFetcher) GenerateSimpleDirPath(repoName string) string {
```

Comment:
I recommend against exposing functions publicly when it can be avoided

Reply:
To test it wouldn't we have to put it in the same package then? Since they are in separate packages I wasn't sure how to make a test unless its exposed

Reply:
You can create a unit test against running Fetch with the new optional field you've created and validate that the generated path matches what you expected.